includes/Plugin.php

```diff
@@ -140,6 +140,7 @@ class Plugin extends ServiceBasedPlugin {
 		'user_preferences'             => User\Preferences::class,
 		'remove_transients'            => Remove_Transients::class,
 		'web_stories_block'            => Block\Web_Stories_Block::class,
+		'speculative_prerendering'     => Admin\Speculative_Prerendering::class,
```

Comment:
```suggestion
		'speculative_prerendering'     => Speculative_Prerendering::class,
```

Let's remove the `Admin` namespace here as this will be used on the frontend as well.